docs/Formhandling.stories.mdx

```diff
@@ -0,0 +1,178 @@
+  const [terms, setTerms] = useState(false);
+
+  const handleSubmit = (e) => {
+    const values = {
+      email: email,
+      password: password,
+      country: country,
+      birthday: birthday,
+      payment: payment,
+      terms: terms
+    };
+    alert(JSON.stringify(values, null, 2));
+    console.log(values);
+    //prevent page reload
+    e.preventDefault();
+  };
+  const handleEmailInput = (e) => {
+    setEmail(e.target.value);
+  };
+  const handlePasswordInput = (e) => {
+    setPassword(e.target.value);
+  };
+  const handleCountryChange = (e) => {
+    setCountry(e.detail.selectedOption.innerText);
+  };
+  const handleBirthdayChange = (e) => {
+    setBirthday(e.detail.value);
+  };
+  const handlePaymentSelectionChange = (e) => {
+    const selected = [];
+    Object.entries(e.detail.items).forEach(([key, val]) => {
+      selected.push(val.text);
+    });
+    setPayment(selected);
+  };
+  const handleTermsChange = (e) => {
+    setTerms(e.target.checked);
+  };
+
+  return (
+    <ThemeProvider>
+      <Form onSubmit={handleSubmit}>
+        <FormGroup titleText="Create Account">
+          <FormItem label={<Label required>Email</Label>}>
+            <Input name="email" required type={InputType.Email} value={email} onInput={handleEmailInput} />
+          </FormItem>
+          <FormItem label={<Label required>Password</Label>}>
+            <Input name="password" required type={InputType.Password} value={password} onInput={handlePasswordInput} />
+          </FormItem>
+          <FormItem label="Country">
+            <Select name="country" onChange={handleCountryChange}>
+              <Option>Germany</Option>
+              <Option>France</Option>
+              <Option>Italy</Option>
+            </Select>
+          </FormItem>
+          <FormItem label="Date of Birth">
+            <DatePicker value={birthday} onChange={handleBirthdayChange}></DatePicker>
+          </FormItem>
+          <FormItem label="Payment methods">
+            <MultiComboBox onSelectionChange={handlePaymentSelectionChange}>
+              <MultiComboBoxItem text="Credit card" />
+              <MultiComboBoxItem text="PayPal" />
+              <MultiComboBoxItem text="Bank transfer" />
+            </MultiComboBox>
+          </FormItem>
+          <FormItem>
+            <CheckBox
+              required
+              name="terms"
+              text="I accept the terms of service"
+              checked={terms}
+              onChange={handleTermsChange}
+            />
+          </FormItem>
+          <FormItem>
+            <Button submits>Submit</Button>
+          </FormItem>
+        </FormGroup>
+      </Form>
+    </ThemeProvider>
+  );
+}
+```
+
+</details>
+
+Now the `RegisterForm` would compile to this:
+
+<RegisterForm />
+
+For the submits property of the `Button` to have effect, you must add the following import to your project: `import '@ui5/webcomponents/dist/features/InputElementsFormSupport.js';`
```

Comment:
The import creates an internal native input component inside the web-component, so that it can be submitted not only via submit button press, but also by pressing the enter key in most components. Basically, the import should enable web-components to behave like native input elements in a form.